src/mintpy/objects/gps.py

```diff
@@ -17,8 +17,13 @@
 import numpy as np
 from pyproj import Geod
 
-from mintpy.objects.coord import coordinate
-from mintpy.utils import ptime, readfile, time_func, utils1 as ut
+from mintpy.utils import (
+    ptime,
+    readfile,
+    time_func,
+    utils0 as ut0,
+    utils1 as ut,
```

Comment:
`utils0` is imported within `util1` already, so we don't need an extra `utils0 as ut0` here. This is a messy part in mintpy, and the [current logic](https://mintpy.readthedocs.io/en/latest/api/module_hierarchy/) is: 1) `utils1` imports everything from `utils0`; 2) `utils` imports everything from `utils0` and `utils1`.